firefox_ui_tests/functional/locationbar/test_star_in_autocomplete.py

```diff
@@ -25,10 +25,13 @@ def setUp(self):
         self.prefs.set_pref(self.PREF_LOCATION_BAR_SUGGEST, 0)
 
     def tearDown(self):
-        FirefoxTestCase.tearDown(self)
+        # Close the autocomplete results
+        self.browser.navbar.locationbar.autocomplete_results.close()
 
-        self.prefs.reset_pref(self.PREF_LOCATION_BAR_SUGGEST)
+        FirefoxTestCase.tearDown(self)
```

Comment:
Better do a try/finally here, so in case of something fails before we call the tearDown of FirefoxTestCase.